subprojects/docs/src/docs/userguide/task_configuration_avoidance.adoc

```diff
@@ -341,14 +377,30 @@ You can solve the following in three ways:
   - **Use `TaskProvider` variable.**
   Useful when the task is referenced multiple time in the same build script.
 +
+====
+[.multi-language-sample]
+=====
 [source,groovy]
 ----
 def someTask = tasks.register "someTask"
 
-task aEagerTask {
+task anEagerTask {
     dependsOn someTask
 }
 ----
+=====
+[.multi-language-sample]
+=====
+[source,kotlin]
+----
+val someTask = tasks.register("someTask")
```

Comment:
:x: can be simplified to `val someTask by tasks.registering` to avoid repeating the task name